tests/unit/test_rules_country_origin.py

```diff
@@ -0,0 +1,122 @@
+from importlib.util import find_spec
+from unittest.mock import call, MagicMock, patch
+
+import pytest
+
+from tests.factories.rule import OriginCountryRuleFactory
+from tests.factories.segment import SegmentFactory
+from wagtail_personalisation.rules import get_geoip_module
+
+
+@pytest.mark.django_db
+def test_get_cloudflare_country_with_header(rf):
+    segment = SegmentFactory(name='Test segment')
+    rule = OriginCountryRuleFactory(segment=segment, country='GB')
+    request = rf.get('/')
+    request.META['HTTP_CF_IPCOUNTRY'] = 'PL'
+    assert rule.get_cloudflare_country(request) == 'pl'
```

Comment:
Is the `test_user` function evaluated in these tests?

Reply:
https://github.com/wagtail/wagtail-personalisation/blob/6b3f90587577d6809723a6c9d33ace83bda978e9/tests/unit/test_rules_country_origin.py#L100-L107

That's true, I don't evaluate what value it returns. Will fix.

Reply:
Hi @jberghoef, I added more test cases with the `test_user` method. Can you look and review again please? Thanks!